plans/tier1.fmf

```diff
@@ -283,3 +283,18 @@ discover+:
   - name: reboot after conversion
     how: ansible
     playbook: tests/ansible_collections/roles/reboot/main.yml
+
+/config_file:
+  adjust:
+      enabled: false
+      when: >
+        distro != centos-7
```

Comment:
Are we running this just for CentOS 7.9, @hosekadam ?

Reply:
@danmyway Yeah, just one system is fine. I choose the centos7, but I think it doesn't matter which one we would run. So we can just wait for the fix you mentioned.

Reply:
Okay then, I would maybe suggest switching centos7 for centos8 at least, as the test suite might run faster there
Other than that, I am fine with that
Great job, thank you!

Reply:
Switched :+1: